lint.sh

```diff
@@ -1 +1,3 @@
-pre-commit run --all-files || pre-commit run --all-files
+poetry run black . --color
+poetry run isort .
+poetry run mypy --config-file=pyproject.toml
```

Comment:
modified this to run the linters on all the files, since it only takes a couple seconds